tests/OTAnalytics/application/use_cases/test_load_otconfig.py

```diff
@@ -34,7 +34,7 @@ def test_load(self, otconfig: OtConfig) -> None:
         clear_repositories = Mock()
         config_parser = Mock()
         config_parser.parse.return_value = otconfig
-
+        remark_repository = Mock()
```

Comment:
Please ensure that the remark is saved in the `remark_repository`:
- Add a remark to `otconfig` fixture in line 30ff
- Verify call to `remark_repository` in line 73ff

The same belongs to the failing test, where the remark is not allowed to be added to the repository